app/src/org/commcare/dalvik/activities/CommCareSetupActivity.java

```diff
@@ -69,6 +59,14 @@
     public static final String KEY_AUTO = "is_auto_update";
     public static final String KEY_START_OVER = "start_over_uprgrade";
     public static final String KEY_LAST_INSTALL = "last_install_time";
+
+    @Override
+    public void OnURLChosen(String url) {

```

Comment:
Sandwiching a method in-between class fields is a bad place for a method. Please move this somewhere below the class constructor.